apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/SessionPoolOptions.cs

```diff
@@ -162,13 +163,32 @@ public TimeSpan Timeout
             set => _timeout = CheckPositiveTimeSpan(value);
         }
 
+        /// <summary>
+        /// The maximum number of sessions that will be created in a batch.
+        /// Sessions created are associated to the gRPC channel they are created on, so
+        /// all sessions created in a batch are associated to the same gRPC channel.
+        /// Batch size should be limited so as not to overload a given channel.
+        /// If the sessions needing to be created at any given time are more than this value
+        /// then multiple batches of this size or less will be created.
+        /// </summary>
+        /// <remarks>
+        /// <para>
+        /// This value must be positive. The default value is 5.
+        /// </para>
+        /// </remarks>
+        public int CreateSessionMaximumBatchSize
+        {
+            get => _createSessionMaximumBatchSize;
+            set => _createSessionMaximumBatchSize = GaxPreconditions.CheckArgumentRange(value, nameof(value), 1, int.MaxValue);
+        }
+
         /// <summary>
         /// The maximum number of session create operations allowed to occur simultaneously per session pool.
         /// Spanner has limits on the number of sessions that can be created concurrently without affecting performance.
         /// This value is not typically changed.
         /// </summary>
         /// <para>
-        /// This value must be positive. The default value is 10.
+        /// This value must be positive. The default value is 50.
```

Comment:
This feels odd to me. If we're using batches, I'd expect us to use *fewer* concurrent operations, not more.

Reply:
Ping on this.

Reply:
Done here and in configuration.md as well.